src/server.js

```diff
@@ -1,168 +1,161 @@
 #!/usr/bin/env node
 
 const fs = require('fs');
-const koa = require('koa');
+const Koa = require('koa');
+const puppeteer = require('puppeteer');
 const tmp = require('tmp');
 const winston = require('winston');
-const createPhantomPool = require('phantom-pool').default;
-
-const logger = new (winston.Logger)({
-    level: process.env.SCREENIE_LOG_LEVEL || 'info',
-    transports: [
-        new (winston.transports.Console)({
-            timestamp: () => new Date().toISOString(),
-        }),
-    ],
-});
 
-const app = koa();
-const pool = createPhantomPool({
-    min: process.env.SCREENIE_POOL_MIN || 2,
-    max: process.env.SCREENIE_POOL_MAX || 10,
+const logger = new winston.Logger({
+  level: process.env.SCREENIE_LOG_LEVEL || 'info',
+  transports: [
+    new winston.transports.Console({
+      timestamp: () => new Date().toISOString()
```

Comment:
Our linters usually require ending comma in object literals, so I'd prefer we keep that here (and on the two lines below this at least.

Reply:
Yeah Ill add the rc file which fixes this.